cmd/minikube/cmd/config/config.go

```diff
@@ -96,12 +97,21 @@ var settings = []Setting{
 var ConfigCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
 	Short: "Modify minikube config",
-	Long:  `config modifies minikube config files using subcommands like "minikube config set vm-driver kvm"`,
+	Long: `config modifies minikube config files using subcommands like "minikube config set vm-driver kvm" 
+Configurable fields: ` + "\n\n" + configurableFields(),
 	Run: func(cmd *cobra.Command, args []string) {
 		cmd.Help()
 	},
 }
 
+func configurableFields() string {
+	var fields []string
+	for _, s := range settings {
+		fields = append(fields, "  "+s.name)
```

Comment:
Is the space there for indentation in the markdown? A "bullet" might make it easier to read.